.github/workflows/build.yml

```diff
@@ -28,12 +28,21 @@ jobs:
       with:
         cache-on-failure: true
         cache-all-crates: true
-    
+            
     - name: Setup git config
       run: |
         git config --global user.name "${GITHUB_ACTOR}"
         git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"
 
+    - name: Rust Setup
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        target: wasm32-unknown-unknown
```

Comment:
Why add `wasm32` target to the CI?

Reply:
Good point - we are not building Substrate here